linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/oval/shared.xml

```diff
@@ -45,6 +51,21 @@
     <ind:pattern operation="pattern match">^[\s]*(?i)ClientAliveCountMax[\s]+([\d]+)[\s]*(?:#.*)?$</ind:pattern>
     <ind:instance datatype="int">1</ind:instance>
   </ind:textfilecontent54_object>
+  {{%- if config_is_distributed == "true" %}}
```

Comment:
```suggestion
  {{%- if sshd_distributed_config == "true" %}}
```

Reply:
Forgot to add this comment to previous review

Reply:
Fixed. Thanks